stable/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 6.3.12
+version: 6.3.13
```

Comment:
Increase minor version instead: `6.4.0`

Reply:
thanks for the review. I increased the minor version, and also took the time to add labels/annotations on the slaves too, to be consistent